chainio/clients/builder.go

```diff
@@ -68,7 +69,13 @@ func BuildAll(
 		return nil, types.WrapError(errors.New("Failed to create Eth WS client"), err)
 	}
 
-	signerV2, addr, err := signerv2.SignerFromConfig(signerv2.Config{PrivateKey: ecdsaPrivateKey}, big.NewInt(1))
+	rpcCtx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
you need to rebase?

Reply:
I don't think so.. why are you saying this?

Reply:
you pushed this in another PR? no? am I reading it wrong https://github.com/Layr-Labs/eigensdk-go/pull/145/files

Reply:
Ah good call, rebased.